doc/threat_model/Untrusted_Capsule_Isolation.md

```diff
@@ -0,0 +1,27 @@
+Untrusted Capsule Isolation
```

Comment:
Should we be saying something about availability here? Currently manual  inspection is required to ensure that capsules do not destroy availability guarantees (maliciously or accidentally) through the use of `panic`, infinite loops, etc.

Reply:
I think there's already an explicit assumption in the capsule design that they're scheduled cooperatively, from before the security model. But it's certainly should be mentioned in the document.

Reply:
Yes, I agree that this doc should discuss availability. Done.